.github/workflows/firebase-hosting-merge.yml

```diff
@@ -1,20 +1,27 @@
-# This file was auto-generated by the Firebase CLI
-# https://github.com/firebase/firebase-tools
-
-# name: Deploy to Firebase Hosting on merge
-# 'on':
-#   push:
-#     branches:
-#       - master
-# jobs:
-#   build_and_deploy:
-#     runs-on: ubuntu-latest
-#     steps:
-#       - uses: actions/checkout@v3
-#       - run: npm ci && npm run build
-#       - uses: FirebaseExtended/action-hosting-deploy@v0
-#         with:
-#           repoToken: '${{ secrets.GITHUB_TOKEN }}'
-#           firebaseServiceAccount: '${{ secrets.FIREBASE_SERVICE_ACCOUNT_UT_DNR_UGS_HAZARDS_DEV }}'
-#           channelId: live
-#           projectId: ut-dnr-ugs-hazards-dev
+name: Deploy to Firebase Hosting on merge
```

Comment:
I uncommented this out and expanded it to only work on merges to 'master' and 'develop'. I tested that the deploy correctly went to the [dev environment](https://ut-dnr-ugs-hazards-dev.web.app/)